app/Http/Controllers/QuestionController.php

```diff
@@ -15,25 +17,34 @@ public function __construct(
 
     }
 
-    public function index(){
-        $res = $this->api
-            ->sendRequest('questions');
+    /**
+     * @throws ExceptionAlias
+     */
+    public function index(): JsonResponse
+    {
+
+        $response = $this->api->sendRequest('questions');
+
+        $data = $response['items'];
+
+        return response()->json(QuestionResource::collection($data));
```

Comment:
👀 👀  resursebi azrs kargaven

Reply:
რატო ?

Reply:
roca areturneb resurss jsonad tavisit transformirdeba exla am shemtxvevashi imushavebs imitoro json( shi obieqtia da eg gardaqmnis mara prosta return QuestionResource::collection($data) c json s daabrunebs, check docs

Reply:
Yes but, like status codes much ? 

Hear me out, Resources are just wrappers for casting. fancy ones, one could argue, but no more.
We need to keep consistency with out API and return response()->json($data, $status);
always. 
In the future, when we wrap the native response with a nice class, wink, wink, it's all gonna be consistent, and heavily reliant on the fact that resources do resolve themselves to arrays cause that is what we want. 

Controllers return JsonResponses, 
Resources allow to shape and format output data using a fancy class.

Reply:
ok I thought we could just like use maybe something like baseResource class or something and include all our wrapper stuff but  I dont know much i guess

Reply:
ნო, გაიხსენე ბეის ექშენი ან ბეის კონტროლერი შეიძლება მარა, ბეის რესურსი არა, რესურსი არის ისეთი როჟა რომელშიც ხელით წერ ვალიუებს, მკასტველია, ყველაზე მანუალ კლასია, და ბევრი აბსტრაქცია ისედაც და ასედაც მაგიტო არ ჭირდება. 

1 კონკრეტული ტასკი აქვს, დამეპოს შემოსული არაი შენ აღწერილ არაიზე და ვსიო.

დანარჩენი მოვალეობები სხვებმა გაინაწილონ